cmd/utils/flags.go

```diff
@@ -1586,6 +1595,13 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	if ctx.GlobalIsSet(AncientFlag.Name) {
 		cfg.DatabaseFreezer = ctx.GlobalString(AncientFlag.Name)
 	}
+	if ctx.GlobalIsSet(AncientRPCFlag.Name) {
+		remote := ctx.GlobalString(AncientRPCFlag.Name)
+		if _, err := url.Parse(remote); err != nil {
```

Comment:
Can we just trust the value fall to `rpc.Dial` and get handled (fail hard, database won't open) if there's an error there?